### PR TITLE
Translate support plans for Ruby 2.0.0 and Ruby 2.1 (pt)

### PR DESCRIPTION
**Pull request by fpgentil**
This PR contains the translation for the latest post `Support plans for Ruby 2.0.0 and Ruby 2.1`.

It is my first PR, I read the **guidelines for contributors** and also tested the translation locally. Please forgive me if I have missed something.

I'll work on the other posts as well, I just decided to open this PR because the post was published today (2016-02-24) and I found it very important.

### CODE REVIEW

**Comment by stomar** [comment]
Welcome @fpgentil, and already hanks for your first contribution!

"Technically" the PR looks pretty good, only objection is that lines should be wrapped
at about 80 characters; I'll point out 2 typos shortly. Preferably you would amend your commit and force push your branch again.

I do not speak or read `pt`, though, so I can not review language.

@fuadsaud, @alanwillms

File: pt/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Favor entre em contato conosco através da lista de email ruby-core caso queira continuar mantendo uma branch da versão 2.0.0 com motivos pelos quais você não pode fazer o upgrade.
+
+## Ruby 2.1
+
+Estamos planejando em lançar uma versão de Ruby 2.1.9 até o final de Março.
+Isso significa que após o lançamento da versão 2.1.9 nós não iremos mais portar nenhuma correção de *bugs* para a versão 2.1, com exceção das correções de segurança.
+
+Nós recomendamos que você comece a planejar o *upgrade* para Ruby 2.3 ou 2.2
+
+A propósito, nós estamos também planejando em lançar a versão 2.1.10 logo após a *relsease* 2.1.9.
```
Comment (stomar) [inline pt/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md]: release

File: pt/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+É altamente recomendado que você faça o upgrade para Ruby 2.3 ou 2.2 o mais rápido possível.
+
+Favor entre em contato conosco através da lista de email ruby-core caso queira continuar mantendo uma branch da versão 2.0.0 com motivos pelos quais você não pode fazer o upgrade.
+
+## Ruby 2.1
+
+Estamos planejando em lançar uma versão de Ruby 2.1.9 até o final de Março.
+Isso significa que após o lançamento da versão 2.1.9 nós não iremos mais portar nenhuma correção de *bugs* para a versão 2.1, com exceção das correções de segurança.
+
+Nós recomendamos que você comece a planejar o *upgrade* para Ruby 2.3 ou 2.2
```
Comment (stomar) [inline pt/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md]: Ruby 2.3 ou 2.2. (missing full stop)

File: pt/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+Anunciamos os futuros planos de suporte para Ruby 2.0.0 e Ruby 2.1.
+
+## Sobre Ruby 2.0.0
+
+Como anunciado anteriormente, todo o suporte para Ruby 2.0.0 se encerra hoje.
+*Bugs* e correções de seguranças de versões mais recentes de Ruby não serão mais portadas à versão 2.0.0 e nenhum *patch release* da versão 2.0.0 será lançado.
+
+É altamente recomendado que você faça o upgrade para Ruby 2.3 ou 2.2 o mais rápido possível.
```
Comment (alanwillms) [inline pt/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md]: `Recomendamos fortemente` para `We highly recommend`, talvez?

File: pt/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Favor entre em contato conosco através da lista de email ruby-core caso queira continuar mantendo uma branch da versão 2.0.0 com motivos pelos quais você não pode fazer o upgrade.
```
Comment (alanwillms) [inline pt/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md]: `Por favor` x `Favor`. A documentação em português do `Git` trata `branch` como palavra masculina, então acho que seria `um branch` ao invés de `uma branch`.

File: pt/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+## Ruby 2.1
+
+Estamos planejando em lançar uma versão de Ruby 2.1.9 até o final de Março.
```
Comment (alanwillms) [inline pt/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md]: Só uma dica: em **p**ortuguês **m**arço pode começar com [letra minúscula](http://www.lpeu.com.br/q/ng23j).

**Comment by alanwillms** [comment]
I wrote a few minor notes.

**Comment by fpgentil** [comment]
@alanwillms @stomar 
Thanks for the quick feedbacks, I updated the PR with the following "fixes"

* Wrap lines in 80 chars
* Fix typos ("relsease")
* Add missing full-stop dot
* Portuguese improvements

File: pt/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md (hunk 5 above)
Comment (fpgentil) [inline pt/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md]: Não sabia disso, muito obrigado :+1:

File: pt/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md (hunk 3 above)
Comment (fpgentil) [inline pt/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md]: Acho que faz mais sentido mesmo, eu pensei em algumas outras alternativas pra tentar usar `Recomendamos` mas não tinha ficado bom, acho que assim (`Recomendamos fortemente`) está bem melhor :smile:

**Comment by stomar** [comment]
@fpgentil Nice work!
@alanwillms Thanks for proofreading!

**Pull request merged**